wcon_format_spec.md

```diff
@@ -195,21 +195,22 @@ _Note: JSON must be unicode, so the micro-symbol must be encoded as unicode.  Th
 
 The following units should be handled:
 
-| Unit       | Abbrevations |
-| ---------- | ------------ |
-| second     | s sec        |
-| minute     | m min        |
-| hour       | h            |
-| day        | d            |
-| metre      | m            |
-| inch       | in           |
-| micron     |              |
-| fahrenheit | F            |
-| centigrade |              |
-| celsius    | C            |
-| percent    | %            |
+| Unit       | Abbreviations |
+| ---------- | ------------- |
+| second     | s sec         |
+| minute     | min           |
+| hour       | h             |
+| day        | d             |
+| metre      | m             |
+| inch       | in            |
+| micron     | *(none)*      |
+| fahrenheit | F             |
+| centigrade | *(none)*      |
```

Comment:
I saw `C` was given as the abbreviation for `celsius`.  But since `Centigrade` is a synonym of `celsius`, `C` could be listed as an abbreviation for `Centrigrade`, as well.

OK.